.github/workflows/publish.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - name: Check module version
         uses: tarantool/actions/check-module-version@master
         with:
-          module-name: 'metrics-export-role'
+          module-name: 'roles.metrics-export'
```

Comment:
Is it works fine? As far as I remember, action uses Tarantool 2.10

Reply:
Thx, changed.
For now reworked this job.
Will open PR in tarantool/actions to add an ability to choose tarantool version.